thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java

```diff
@@ -624,10 +624,12 @@ private static void handleException(
 
         final TBase<?, ?> result = func.newResult();
         final HttpData content;
-        if (func.setException(result, cause)) {
+        final Throwable peeledException = Exceptions.peel(cause);
```

Comment:
How about peeling a bit early?
Here https://github.com/line/armeria/blob/master/thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java#L591
and here:
https://github.com/line/armeria/blob/master/thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/THttpService.java#L594